### PR TITLE
find and replace of auto to its correct type

### PR DESCRIPTION
**Pull request by erikzenker**
I replaced all occurrence of auto by its correct type, otherwise the
CUDA benchmark were not able to compile.

### CODE REVIEW

File: src/par/Cuda.cu [hunk 1]
```diff
@@ -407,7 +407,7 @@
 
             if(insideC)
             {
-                auto const CIdx1d(gridThreadIdxY*ldc + gridThreadIdxX);
+	        TSize const CIdx1d(gridThreadIdxY*ldc + gridThreadIdxX);
```
Comment (BenjaminW3) [inline src/par/Cuda.cu]: You introduced a tab in here

File: src/par/Cuda.cu [hunk 2]
```diff
@@ -111,7 +111,7 @@
...
```
Comment (BenjaminW3) [inline src/par/Cuda.cu]: You introduced a tab in here

**Comment by BenjaminW3** [comment]
With which CUDA version have you been building? Why are there errors with auto?

**Comment by erikzenker** [comment]
I was compiling with the following cluster configuration:
- gcc/4.9.2
- boost
- cmake/3.3.0
- cuda/7.0

The error occurs maybe because c++11 is not set for the CUDA benchmark ? I did not investigate this to much because the types hidden by auto where not to complicated. Writing TSize instead of auto is no big deal.

File: src/par/Cuda.cu (hunk 1 above)
Comment (erikzenker) [inline src/par/Cuda.cu]: fixed

File: src/par/Cuda.cu (hunk 2 above)
Comment (erikzenker) [inline src/par/Cuda.cu]: fixed

**Pull request merged**